peps/pep-9999.rst

```diff
@@ -0,0 +1,199 @@
+PEP: 9999
+Title: C API Working Group Charter
+Author: Guido van Rossum <guido@python.org>,
+        Petr Viktorin <encukou@gmail.com>,
+        Victor Stinner <vstinner@python.org>,
+        Steve Dower <steve.dower@python.org>,
+        Irit Katriel <irit@python.org>
+Status: Draft
+Type: Process
+Topic: Typing
+Created: 11-Oct-2023
+Python-Version: 3.13
+
+Abstract
+========
+
+This PEP proposes to establish the C API Working Group:
+a small committee of Python core developers responsible for
+maintaining and developing the Python C API.
+
+The working goup will maintain
+documentation, test suites and tooling related to Python's C API.
+It is the deciding body for changes to the C API,
+from the addition or removal of individual API functions, types, etc.,
+to the the acceptance of new designs of a more or less radical nature.
+
+The working goup's mandate is to represent the interests of all Python users,
+but especially all maintainers of code that uses Python's C API,
+whether in the context of CPython, using an alternate Python implementation,
+or using a binding framework for other languages.
+
+The working group serves at the pleasure of the Python Steering Council.
+The working group's members are the listed authors of this PEP.
+This document serves as the working group's charter.
+
+Epigraph
+========
+
+.. |br| raw:: html
+
+   <br />
+
+KEEPER
+|br|
+Stop!
+Who would cross the Bridge of Death must answer me these questions three,
+ere the other side he see.
+
+#. Who invented Python?
+#. In what year did Python 2.7 reach its End-of-Life?
+#. What is the future of the Python C API?
+
+LANCELOT
+|br|
+Auuuuuuuugh!
```

Comment:
There's [ReST for this](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks):
```suggestion
| KEEPER
| Stop!
  Who would cross the Bridge of Death must answer me these questions three,
  ere the other side he see.

#. Who invented Python?
#. In what year did Python 2.7 reach its End-of-Life?
#. What is the future of the Python C API?

| LANCELOT
| Auuuuuuuugh!
```

Reply:
Thanks, I committed a slight variation. I asked ChatGPT how to do this and it gave me the approach that I used (and several others, but that was the only one that seemed to work).